apps/accounts/models.py

```diff
@@ -148,6 +148,12 @@ class UserProfile(models.Model):
             'Check this to allow the account to authorize applications.'),
     )
 
+    mfa_login_mode = models.CharField(default="",
+                                      choices=[('', 'None'), ('EMAIL', 'Email'), ('SMS', 'Text Message (SMS)')],
+                                      blank=True,
+                                      null=True,
+                                      max_length=5)
+
```

Comment:
these un-applied changes are left so that the corresponding tables removal and fields alterations not performed in DB, thinking that might also be the considerations going on around until 04-20-2020, the fhir_source field removal from crosswalk (a FK) only happened at model layer, other invitations stuff are never used - those tables are empty at SBX and PROD..., but there are some references in the permissions system which still refer to those invitations action..., the current way is to set level of migrations - i.e. - going forward, you can showmigrations/makemigrations/migrate on a new starting point, those un-used tables and fields are left there, only migrations files are squashed (consolidated).

if we want to go with DB cleansing - un-used tables removed (not much concern there on performance since those invitations are not used), and some fields altered - most significant is: crosswalk.fhir_source (FK), queried on SBX and PROD, there are 712 crosswalk records with fhir_source value (SBX), and 16111 records with fhir_source values on PROD, basically, it is alter table column on a table of 100K ish rows, I tried migrate with a local DB with 30k crosswalk, it did not take a lot time, but we can experiment on a PROD clone, if it takes long enough, then we might want to do it at a low traffic time.